README.md

```diff
@@ -1,2 +1,17 @@
 # hw01
 Repository for first CFSS homework assignment
+
+#About Me
```

Comment:
When adding headers, add a space between the hashtag symbol and the header title. Like

```
# About Me
```